extensions/datasource/deployment-spi/src/main/java/io/quarkus/datasource/deployment/spi/DevServicesDatasourceContainerConfig.java

```diff
@@ -41,6 +43,7 @@ public DevServicesDatasourceContainerConfig(Optional<String> imageName,
         this.username = username;
         this.password = password;
         this.initScriptPath = initScriptPath;
+        this.showLogs = showLogs;
```

Comment:
Minor nitpick, but move this to the last line to match the order

Reply:
Also don't forget to `git rebase main` before you force push and confogure to rebase on pull: https://github.com/quarkusio/quarkus/blob/main/CONTRIBUTING.md#before-you-contribute